services/hugo_service.py

```
@@ -63,10 +67,17 @@ def start(self, debug=False):
                 self.server_url,
                 "--disableFastRender",
             ]
-            if debug:
-                cmd.append("-D")
-            # Docker 部署时禁用了 Hugo Modules，需要显式指定主题
+            # 管理后台预览服务器默认渲染草稿，方便查看新建文章
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)